### PR TITLE
JBPM-8114 - Add node instance badge counter in process diagram

### PR DESCRIPTION
**Pull request by cristianonicolai**
Depends on https://github.com/kiegroup/appformer/pull/612

### CODE REVIEW

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramViewImpl.java [hunk 1]
```diff
@@ -142,6 +159,71 @@ public void hideNodeActions() {
...
+        if(svgRect.getWidth() == 0 && svgRect.getHeight() == 0){
+            //SVG not visible
+            return;
+        }
+
+        badges.forEach((nodeId, count) -> {
+            final D3 node = d3.select("#processDiagramDiv svg [bpmn2nodeid=" + nodeId + "]");
+            D3.DOMRect bb = node.node().getBoundingClientRect();
+            final D3 group = node.append("g")
+                    .attr("transform", "translate( " + bb.getWidth() + ", " + bb.getHeight() + ")")
```
Comment (nmirasch) [inline jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/ProcessInstanceDiagramViewImpl.java]: @cristianonicolai I see the badge placement a bit weird over some components like start or split
processinstancebadge
I tried to see if bb.getWidth()/2 but not sure if it looks better or if it is further of the UX proposed design, what do you think?
processinstancebadge2

**Comment by cristianonicolai** [comment]
@nmirasch thanks, your suggestion looks better indeed, applied that.

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Comment by cristianonicolai** [comment]
Jenkins execute full downstream build

**Pull request merged**